### DISCUSSION TITLE
Add class names to the rendered SVGs

### DISCUSSION THREAD

**Discussion by lannka** [CoderLine/alphaTab#613] [Ideas]
I'm wondering if it is feasible to add CSS classes to those SVGs, for example the title section, the chord diagram section, the turning, etc. They can be used to hide content as needed using pure CSS.

**Comment by lannka** [comment]
I found that this is already well supported via setting: https://www.alphatab.net/docs/reference/settings/notation/elements/#description

**Reply by Danielku15** [reply]
Yes, and I recommend using these options to show/hide elements. It requires a re-rendering of the music sheet but it should be quite fast. The problem with dynamic changes via CSS is the fact that alphaTab relies on some relative positions within the container to position elements like the cursor and react on user interaction (clicks). Even though the rendering is possible via SVG you rather have to consider it like a rendering canvas (like HTML5 is also supported). 

I guess some optimizations would be possible but they also have to be maintained and developed. That said I recommend using the settings you found with a re-rendering trigger.
